docs/formats.rst

```diff
@@ -735,13 +735,13 @@ Example file:
 .. versionadded:: 2.3
 
 .NET Resource (.resx) file is a monolingual XML file format used in Microsoft
```

Comment:
Not sure here. Only thing I could find was "VB.NET resource"… ".NET Applications" also seems iffy.

Reply:
@nijel

Reply:
It seems to be called just "XML resource files", see https://docs.microsoft.com/cs-cz/dotnet/framework/resources/working-with-resx-files-programmatically. I've followed translate-toolkit naming here: http://docs.translatehouse.org/projects/translate-toolkit/en/latest/formats/resx.html

Reply:
renamed and resolved